docs/content/installation/csi_ug_install_operator_openshift.md

```diff
@@ -0,0 +1,47 @@
+# Installing the driver with the OpenShift web console
+
+When using the Red Hat® OpenShift® Container Platform, the operator for IBM® block storage CSI driver can be installed directly from OpenShift web console, through the OperatorHub. Installing the CSI (Container Storage Interface) driver is part of the operator installation process.
+
+The Red Hat OpenShift Container Platform uses the following `SecurityContextConstraints` for the following `serviceAccounts`:
+
+**Note:** This data is for informational purposes only.
+
+|serviceAccount|SecurityContextConstraint|
+|--------------|-------------------------|
+|ibm-block-csi-operator|restricted|
+|ibm-block-csi-controller-sa|anyuid|
+|ibm-block-csi-node-sa|privileged|
+
+1. From Red Hat OpenShift Container Platform **Home** > **Projects**, click **Create Project**. In the **Create Project** dialog box, enter a Project name (also referred to as namespace). Click **Create** to save.
+
+2. From **Operators** > **OperatorHub**. Select the namespace from `Projects:<namespace>`, as defined in step 1.
+
+3. Search for IBM block storage CSI driver.
+
+4. Select the **Operator for IBM block storage CSI driver** and click **Install**. <br /><br />The **Operator Installation** form appears.
```

Comment:
since 1.7.0, the display name of the operator [was changed](https://github.com/IBM/ibm-block-csi-operator/pull/198).
this requires alignment across our docs

Reply:
I'll update within the latest release

Reply:
fixed in https://github.com/IBM/ibm-block-csi-driver/pull/466